Library/Formula/svtplay-dl.rb

```diff
@@ -22,6 +15,7 @@ class SvtplayDl < Formula
   end
 
   depends_on "rtmpdump"
+  depends_on "python"
```

Comment:
We don't set mandatory Homebrew Python dependencies, I'm afraid. It'll expect to use our Python, which means everyone has to install our Python, which isn't particularly fun or kind on a system that already keeps a pretty up-to-date Python around.

I guess you can add `depends_on :python => :optional` if @tdsmith isn't opposed, but I'm not sure on the upside here.